tardis/agents/siteagent.py

```diff
@@ -31,6 +31,10 @@ def handle_response(
     ):
         return NotImplemented
 
+    @property
+    def drone_heartbeat_interval(self) -> int:
```

Comment:
It would be nice to add some docstring here.

Reply:
I would rather like to put the docstring to https://github.com/MatterMiners/tardis/blob/e4e35068c1c083a40ef0f0a878bd9f5c93e6545f/tardis/interfaces/siteadapter.py#L150 since it is the base class and contains all docstrings.

Reply:
I don't see any docstring in the siteadapter interface yet. Do we better want to postpone?

Reply:
Damn! I simply forgot to add it.  This is not a big deal. I will add now.